src/components/navbar/side-menu-content.js

```diff
@@ -1,11 +1,17 @@
 export const sideMenuContent = [
-  {
-    menuItem: { caption: 'Dashboard', icon: 'block layout', link: 'dashboard' }
-  },
+  { menuItem: { caption: 'Dashboard', icon: 'block layout', link: 'dashboard' } },
 
   {
     menuItem: { caption: 'Resellers', icon: 'users', link: 'resellers' }
     // subMenu: ['All users', 'Registration', 'Download user report']
-  }
+  },
 
+  { menuItem: {
+    caption: 'Log out',
+    icon: 'log out',
+    callback: (navbar) => {
+      navbar.props.logout();
+      navbar.props.history.push('login');
+    }
+  } }
 ];
```

Comment:
Does everything on the sidebar also work on the minified sidebar? I dont think I saw any changes to that file